Atomic/atomic.py

```diff
@@ -30,6 +41,22 @@
 from docker.errors import NotFound
 
 IMAGES = []
+ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
+
+OSTREE_OCIIMAGE_PREFIX = "ociimage/"
+SYSTEMD_UNIT_FILES_DEST = "/etc/systemd/system"
+SYSTEMD_UNIT_FILE_DEFAULT_TEMPLATE = """
+[Unit]
+Description=$NAME
+
+[Service]
+ExecStart=/bin/runc start $NAME
+Restart=on-failure
+WorkingDirectory=$DESTDIR
```

Comment:
Should this be Restart=on-crash

On-Failure will continue to attempt to restart if their is a config error, while On-Crash will restart is runc crashes for some reason.